x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -274,6 +288,15 @@ static CsvSpecReader.CsvTestCase convertToRemoteIndices(CsvSpecReader.CsvTestCas
         return testCase;
     }
 
+    static String remoteIndices(String localIndices) {
+        String[] parts = localIndices.split(",");
+        return Arrays.stream(parts).map(index -> "*:" + index.trim()).collect(Collectors.joining(","));
+    }
+
+    static boolean canUseRemoteIndicesOnly() {
+        return dataLocation == DataLocation.REMOTE_ONLY && Clusters.bwcVersion().onOrAfter(Version.V_9_1_0);
```

Comment:
Worth a comment I think.

Reply:
Sure, I added a comment in https://github.com/elastic/elasticsearch/pull/128111/commits/a8df46a5e17a9d79efea86ccec7ef1741c447e56